ledger/ledger-runner-common/src/test/lib/com/daml/ledger/runner/common/ArbitraryConfig.scala

```diff
@@ -181,9 +182,15 @@ object ArbitraryConfig {
     maxUsersPageSize = maxUsersPageSize,
   )
 
+  def leewayOptionsGen: Gen[LeewayOptions] = {
+    for {
+      l <- Gen.listOfN(4, Gen.option(Gen.posNum[Long]))
+    } yield LeewayOptions(l(0), l(1), l(2), l(3))
+  }
```

Comment:
That would read better even though not as concise as your solution ;)

```suggestion
  def leewayOptionsGen: Gen[LeewayOptions] = {
    for {
      leeway <- Gen.option(Gen.posNum[Long])
      expiresAt <- Gen.option(Gen.posNum[Long])
      issuedAt <- Gen.option(Gen.posNum[Long])
      notBefore <- Gen.option(Gen.posNum[Long])
    } yield LeewayOptions(
      leeway = leeway,
      expiresAt = expiresAt,
      issuedAt = issuedAt,
      notBefore = notBefore
    )
  }
```